Formula/clojure-lsp.rb

```diff
@@ -31,7 +31,9 @@ def install
     require "Open3"
 
     print "testing"
+    system "file", "#{bin}/clojure-lsp"
     stdin, stdout, _, wait_thr = Open3.popen3("#{bin}/clojure-lsp")
```

Comment:
Try redirecting `stderr` to `stdout`. It may be that the output is just being sent there, and that's why you're not picking it up.

Reply:
I changed `stdout.gets` to `stderr.gets`. I'm not sure if that's what you meant - if you have any examples at hand, that would be great.

Reply:
Never mind, we just need to get what's coming to strerr for debugging purposes, will do it